index.html

```diff
@@ -0,0 +1,144 @@
+                    <div>    
+                        <h3 class="item__title">Inovative Ideas</h3>
+                        <p class="item__description">Quisque luctus, quam eget molestie commodo, lacus purus cursus purus, nec rutrum tellus dolor id lorem.</p>
+                    </div>
+                    </div>
+                <div class="content-item">
+                    <img src="img/Services/services-item-3.svg" class="item__icon"></img>
+                    <div>    
+                        <h3 class="item__title">Love Is The Answer</h3>
+                        <p class="item__description">Nulla sed nunc et tortor luctus faucibus. Morbi at aliquet turpis, et consequat felis. Quisque condimentum.</p>
+                    </div>
+                </div>
+                <div class="content-item">
+                    <img src="img/Services/services-item-4.svg" class="item__icon"></img>
+                    <div>    
+                        <h3 class="item__title">Responsive Layout</h3>
+                        <p class="item__description">Sed porttitor placerat rhoncus. In at nunc tellus. Maecenas blandit nunc ligula. Praesent elit leo.</p>
+                    </div>
+                </div>
+                <div class="content-item">
+                    <img src="img/Services/services-item-5.svg" class="item__icon"></img>
+                    <div>    
+                        <h3 class="item__title">24 / 7 Support</h3>
+                        <p class="item__description">Vivamus vel quam lacinia, tincidunt dui non, vehicula nisi. Nulla a sem erat. Pellentesque egestas venenatis lorem .</p>
+                    </div>
+                </div>
+                <div class="content-item">
+                    <img src="img/Services/services-item-6.svg" class="item__icon"></img>
+                    <div>    
+                        <h3 class="item__title">Feel Like A Star</h3>
+                        <p class="item__description">Quisque hendrerit purus dapibus, ornare nibh vitae, viverra nibh. Fusce vitae aliquam tellus.<p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="portfolio main-section" id="portfolio">
+            <div class="porfolio__header">
+                <h2 class="section-title">Portfolio</h2>
+                <p class="section-description">Duis mollis, est non commodo luctus, nisi erat porttitor ligula, eget lacinia odio sem nec elit.</p>
+                <div class="categories">
+                    <a href="##" class="categories__item active_item" id="all">All</a>
+                    <a href="##" class="categories__item" id="web_design" role="button">Web Design</a>
+                    <a href="##" class="categories__item" id="graphic_design" role="button">Graphic Design</a>
+                    <a href="##" class="categories__item" id="artwork" role="button">Artwork</a>
```

Comment:
you can keep in this task, but in general, if you need a button you use <button> not <a> with role. it is not good from an accessibility point of view. and it is easier to handle actions with buttons